shared/components/ResourceGraph/buildGraph/buildNetworkGraph.js

```diff
@@ -139,15 +146,21 @@ export function buildNetworkGraph({ store }, config) {
           const podId = multiplePods
             ? 'composite-pod'
             : store['Pod'][0].metadata.uid;
-          if (!!deployment && store['Pod']?.length) {
-            currentLayer.forEach(svc => {
+          const subscriptionId = store['Subscription'].map(
+            subscription => subscription.metadata.uid,
+          );
+          currentLayer.forEach(svc => {
+            if (!!deployment && store['Pod']?.length) {
               strEdges.push(
                 makeEdge(svc.metadata.uid, podId, {
                   lhead: `cluster_${deployment.metadata.uid}`,
                 }),
               );
-            });
-          }
+            }
+            subscriptionId.forEach(subscription =>
```

Comment:
(Considering renaming mentioned before)
```suggestion
            (subscriptionIds || []).forEach(subscription =>
```